ldbc_sample/json/LdbcSample.graph.yml

```diff
@@ -0,0 +1,7 @@
+vertices:
+  - Person.vertex.yml
+prefix: /tmp/graphar/ldbc_sample/
```

Comment:
It's better not set the prefix of the graph.yml (default is the relative path of the graph yaml) since the test data may not download to the the /tmp/graphar/ldbc_sample/